_data/simple-icons.json

```diff
@@ -4922,6 +4922,11 @@
             "hex": "ED3237",
             "source": "https://lionairthai.com/en/"
         },
+        {
+            "title": "Lit",
+            "hex": "324FFF",
+            "source": "https://lit.dev/images/flame-favicon.svg"
```

Comment:
Could you update the source for this to https://github.com/lit/lit.dev/blob/5e59bdb00b7a261d6fdcd6a4ae529e17f6146ed3/packages/lit-dev-content/site/images/flame-favicon.svg - as that's the source-code of the website. Should allow us better tracking should they change / remove this file in the future!

Reply:
Thank you for your time. Learned something new. :) I have several additional icons incoming, so this review is valuable to me.
I changed the source and jumped into Illustrator to compound those paths. I was something related to my tooling. Now the icon should be even smaller. Linting and tests pass.